astropy/tests/plugins/display.py

```diff
@@ -0,0 +1,142 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+This plugin provides customization of the header displayed by pytest for
+reporting purposes.
+"""
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import os
+import sys
+import datetime
+import locale
+import math
+from collections import OrderedDict
+
+from ...extern import six
+
+from ..helper import ignore_warnings
+from ...utils.introspection import resolve_name
+
+
+PYTEST_HEADER_MODULES = OrderedDict([('Numpy', 'numpy'),
+                                     ('Scipy', 'scipy'),
+                                     ('Matplotlib', 'matplotlib'),
+                                     ('h5py', 'h5py'),
+                                     ('Pandas', 'pandas')])
+
+# This always returns with Astropy's version
+from ... import __version__
+TESTED_VERSIONS = OrderedDict([('Astropy', __version__)])
+
+
+def pytest_report_header(config):
+
+    try:
+        stdoutencoding = sys.stdout.encoding or 'ascii'
+    except AttributeError:
+        stdoutencoding = 'ascii'
+
+    if six.PY2:
```

Comment:
do we need this? astropy 3.0 won't even start up with python2

Reply:
Nope, it's just an oversight since this PR predates the Py3 migration.